### DISCUSSION TITLE
Feature Request: user changeable key mapping for capture/free mouse

### DISCUSSION THREAD

**Discussion by Alexco500** [86Box/86Box#4272] [Ideas]
I am using 86Box on my Mac, and the key combination to free the mouse is not favourable on a Laptop keyboard. So it would be great if it could be changed via command line args or config file settings.

**Comment by cold-brewed** [comment]
If you're on a laptop keyboard, Ctrl+Fn+Right Arrow will work. 

This is because Fn+Right Arrow natively maps to the end key on macOS. Therefore Ctrl+Fn+Right Arrow will send Ctrl+End to release the mouse.

**Reply by Alexco500** [reply]
I didn't know that. And yes, that works, thanks!

**Reply by ylluminate** [reply]
Update: https://github.com/86Box/86Box/discussions/4570#discussioncomment-13690979

**Comment by jriwanek** [comment]
As Covered in $4570, implemented
